js/app.js

```diff
@@ -23,3 +26,27 @@ thumbnailData.forEach(function(ele) {
 projects.forEach(function(p){
   $('#projects').append(p.toHtml());
 });
+
+// Initial Hide
+handleInitialHide = function() {
+  $('#projects').hide();
+};
+
+// Switch Views
+handleMainNav = function() {
+  $('.mainNav').on('click', '.tab', function(){
+    var val = $(this).attr('data-content');
+    $('.tab-content').hide();
+    $('.tab-content').each(function(index){
```

Comment:
There's a better way to select ids in jQuery. Consider how you'd select an id in CSS. If you do that, you don't need to bother with the loop either.